app/views/home/_social.html.erb

```diff
@@ -1,7 +1,22 @@
 <div id="social">
   <% if Rails.env == "production" %>
-  <div class="fb-like" data-send="false" data-layout="button_count" data-width="50" data-show-faces="false" style="vertical-align: middle; bottom: 6px;margin-right:10px" ></div>
-  <a style="width: 60px;height: 20px;bottom: 7px;" href="https://twitter.com/share" class="twitter-share-button">Tweet</a>
-  <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src="//platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
+  <div class="fb-share-button" data-href="https://publiclab.org<%= path %>" data-layout="button_count" style="vertical-align: middle; bottom: 6px;margin-right:10px">Fb Share</div>
```

Comment:
Could we make some code to ensure there is a fall-back? So, perhaps we look to see if there's a @node defined, and use that, but if nothing is provided in `@node` or `path` we display nothing? Thanks!